Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/App.xaml.cs

```diff
@@ -35,6 +35,10 @@ protected override void OnStart()
             if (!AppCenter.Configured)
             {
                 AppCenter.LogLevel = LogLevel.Verbose;
+                if (Current.Properties.ContainsKey(EntryCellTextChanged.UserIdKey) && Current.Properties[EntryCellTextChanged.UserIdKey] is string id)
+                {
+                    AppCenter.SetUserId(id);
```

Comment:
This was probably not tested on Android, the app needs to be configured.

```
 E/AppCenter: AppCenter must be configured from application, libraries cannot use call setUserId.
```

It needs to be moved after AppCenter.Start